docs/lib/predictions/fragments/ios/identify-text.md

```diff
@@ -35,6 +54,29 @@ func detectText(_ image: URL) {
 }
 ```
 
+To get results that utilize on-device capabilities (Core ML), without combining results from the backend, you can use the following to pass into the `options` argument of the `Amplify.Predictions.identify` function.
+```swift
+let options = PredictionsIdentifyRequest.Options(defaultNetworkPolicy: .offline, pluginOptions: nil)
+```
+
+Note that bounding boxes in `IdentifyTextResult` are returned as ratios.  If you would like to place bounding boxes on individual recognized words that appear in the image, use the following method to calculate a frame for a single bounding box.
+```swift
+let image = UIImage(...)
+self.imageView = UIImageView(image: self.image)                        
+...
```

Comment:
Let's try to avoid un-runnable code.  Can we put a comment here to educate the user what kind of code will be in this area?

Reply:
Hmm.. seems pretty clear to me.  Do you have any proposals as to what would make it clearer?

Reply:
As a user, copy and pasting this content would also require me to manually remove multiple lines of ... just to run the code quickly to verify if it’s what I’m looking for. At least move the ... behind a comment

Reply:
Good point, but maybe the bigger question is that maybe we should remove all of this content and leave it just for a sample app?  Who cares about bounding rectangles?

Reply:
removing, we'll figure out a place to put this info, since this is common to multiple places in predictions.